packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -1549,6 +1561,14 @@ class CollectSameUntilThresholdRound(CollectionRound, ABC):
     collection_key: str
     selection_key: Union[str, Tuple[str, ...]]
 
+    required_class_attributes: Tuple[str, ...] = (
+        "done_event",
+        "no_majority_event",
+        "none_event",
+        "collection_key",
+        "selection_key",
```

Comment:
Let's define constans for these, here in base, so that we avoid repeating them everywhere else.